src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mens = people.filter(
+    men => century
+      ? men.sex === 'm' && Math.ceil(men.died / 100) === century
+      : men.sex === 'm'
+  );
```

Comment:
```suggestion
  const mens = people.filter(century
      ?  men => men.sex === 'm' && Math.ceil(men.died / 100) === century
      :  men =>  men.sex === 'm'
  );
```
таким образом проверка будет делаться только один раз